docs/user_guide/hopsworks/search.rst

```diff
@@ -0,0 +1,70 @@
+Search
+========================
+
+The search on the navigation bar can be used to search for Datasets, Feature groups, Features and Training datasets.
+Depending on the page you are in, landing page or project dashboard, the search scope will cover all projects or the specific project you are in.
```

Comment:
The search bar behaves differently depending on whether you are on the landing page or inside a project. When you are on the landing page, you can search for datasets and features in all